service/http.proxy/src/main/java/org/kaazing/gateway/service/http/proxy/HttpProxyServiceHandler.java

```diff
@@ -36,35 +37,82 @@
 import org.slf4j.LoggerFactory;
 
 import java.net.URI;
+import java.util.Collection;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.TreeSet;
+import java.util.regex.Pattern;
 
 import static org.kaazing.gateway.transport.http.HttpHeaders.HEADER_CONNECTION;
+import static org.kaazing.gateway.transport.http.HttpHeaders.HEADER_LOCATION;
+import static org.kaazing.gateway.transport.http.HttpHeaders.HEADER_SET_COOKIE;
 import static org.kaazing.gateway.transport.http.HttpHeaders.HEADER_UPGRADE;
 import static org.kaazing.gateway.transport.http.HttpHeaders.HEADER_VIA;
-import static org.kaazing.gateway.transport.http.HttpStatus.INFO_SWITCHING_PROTOCOLS;
 import static org.kaazing.gateway.transport.http.HttpStatus.CLIENT_NOT_FOUND;
+import static org.kaazing.gateway.transport.http.HttpStatus.INFO_SWITCHING_PROTOCOLS;
 
 class HttpProxyServiceHandler extends AbstractProxyAcceptHandler {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger("http.proxy");
+    private static final Logger LOGGER = LoggerFactory.getLogger("service.http.proxy");
```

Comment:
This looks good.  Is this the convention we are following in general for logging names on services?